dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panel.tsx

```diff
@@ -95,8 +109,34 @@ export const ConfigPanel = ({ visualizations, setCurVisId, callback }: any) => {
     []
   );
 
+   // To check, If user empty any of the value options
+   const isValidValueOptionConfigSelected = useMemo(() => {
+    const valueOptions = vizConfigs.dataConfig?.valueOptions;
+    const { Bar, Line, Histogram, Pie, TreeMap, Gauge, HeatMap } = visChartTypes;
+    const isValidValueOptionsXYAxes = [Bar, Line, Histogram, Pie].includes(curVisId) &&
```

Comment:
Maybe move to the constants file, and name it to something like VIZ_CONTAIN_XY_AXIS? Just thinking other code pieces may also refer to this list, and visualizations that have x/y axes would be added and modified.

Reply:
Thanks, @mengweieric 

As you suggested we have added this list in the constant file with the name VIZ_CONTAIN_XY_AXIS.